app/controllers/journal_entries_controller.rb

```diff
@@ -0,0 +1,50 @@
+class JournalEntriesController < ApplicationController
```

Comment:
please study about `callbacks` and see if we can use any of them in our code (Hint: `before_action`). Resources:
- [Active Record Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html)
- [Filters](https://guides.rubyonrails.org/action_controller_overview.html#filters )

Reply:
I see what yoou mean, but until now i have not implemented a login, so this callbacks are useless, no?

Reply:
There are many other use cases! 
`@journal = Journal.find(params[:journal_id])`
This line is repetitive in our current implementation, we can DRY that code using `before_action`.

Reply:
Ahhh got it!